css-display-3/display-contents-first-line-001.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Display: display:contents and ::first-line</title>
```

Comment:
(This test seems to fail in Gecko, though that seems like a bug in Gecko. Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1305951 )